src/infer_response.cc

```diff
@@ -38,14 +38,20 @@ namespace triton { namespace core {
 //
 Status
 InferenceResponseFactory::CreateResponse(
-    std::unique_ptr<InferenceResponse>* response) const
+    std::unique_ptr<InferenceResponse>* response)
 {
   response->reset(new InferenceResponse(
       model_, id_, allocator_, alloc_userp_, response_fn_, response_userp_,
-      response_delegator_));
+      response_delegator_, response_cnt_
```

Comment:
Pick one of `response_cnt` or `seq_num` and use it in both places - it's harder to follow the flow of this variable with the name difference

Reply:
Personally I think it makes sense to use different names. ResponseFactory keeps track of `response_cnt` which increases monotonically and each `Response` keeps its `seq_num`.

Reply:
Updated in the new commit -- response_index_ and index_ respectively.

Reply:
Reverted the change due to potential breakage in Square Backend.